lib/workers/pr/body/index.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { getName, platform } from '../../../../test/util';
+import { getConfig } from '../../../config/defaults';
+import type { RenovateConfig } from '../../../config/types';
+import { getPrBody } from '.';
+
+jest.mock('./config-description', () => ({
+  getPrConfigDescription: jest.fn(() => ''),
+}));
+jest.mock('./controls', () => ({
+  getControls: jest.fn(() => ''),
+}));
```

Comment:
```suggestion
jest.mock('./config-description');
jest.mock('./controls');
```

just use default mocks jests will do it's magic 🧙